plugin-core/src/main/java/appland/notifications/AppMapNotifications.java

```diff
@@ -160,6 +165,44 @@ public static void showNavieUnavailableNotification(@NotNull Project project) {
         });
     }
 
+    @SuppressWarnings({"removal", "DialogTitleCapitalization"})
+    public static void showWebviewProxyBrokenWarning(@NotNull Project project) {
+        EdtInvocationManager.invokeLaterIfNeeded(() -> {
+            var action = Messages.showDialog(
+                    project,
+                    AppMapBundle.get("notification.brokenProxySupport.content"),
+                    AppMapBundle.get("notification.brokenProxySupport.title"),
+                    new String[]{
+                            AppMapBundle.get("notification.brokenProxySupport.confirmAction"),
+                            AppMapBundle.get("notification.brokenProxySupport.showLinkAction"),
+                    },
+                    0,
+                    Messages.getWarningIcon(),
+                    new DialogWrapper.DoNotAskOption.Adapter() {
+                        @Override
+                        public void rememberChoice(boolean isSelected, int exitCode) {
+                            AppMapApplicationSettingsService.getInstance().setShowBrokenProxyWarning(!isSelected);
+                        }
+                    }
+            );
+
+            if (action == 1) {
+                BrowserUtil.open("https://appmap.io/status");
```

Comment:
I think we should drop this. This page makes no mention of the issue described in the dialog. Let's just offer a 'close' button, the dialog content is sufficient.

Reply:
@dustinbyrne I've rebased and removed the button to open the status page.
![image](https://github.com/getappmap/appmap-intellij-plugin/assets/11473063/5b7f45f1-ec40-4239-b97c-16c9afac3925)